transactor/src/main/scala/io/mediachain/copycat/TransactorService.scala

```diff
@@ -440,22 +441,22 @@ object TransactorService {
   }
   
   def journalBlockReferenceToEvent(ref: Reference): JournalEvent = {
-    JournalEvent().withEvent(Event.JournalBlockEvent(refToRPCMultihashRef(ref)))
+    JournalEvent(JournalEvent.Event.JournalBlockEvent(refToRPCMultihashRef(ref)))
```

Comment:
I get that they're supposed to be analogous but I prefer not to have them completely identically named